src/librustc_trans/back/link.rs

```diff
@@ -887,6 +891,10 @@ fn link_natively(sess: &Session, dylib: bool,
     }
     cmd.args(&sess.target.target.options.post_link_args);
 
+    if needs_sanitizers_runtime_deps {
+        link_sanitizers_runtime_deps(sess, &mut cmd);
```

Comment:
Yeah the `Linker` trait should be available in the block above this, and that should probably be used over frobbing the `Command` directly to one day work on MSVC as well as GNU linkers.